Sources/EpoxyCore/Views/PlatformView.swift

```diff
@@ -0,0 +1,31 @@
+// Created by Cal Stephens on 6/26/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+import SwiftUI
+
+#if canImport(UIKit)
+import UIKit
+
+public typealias UIViewOrNSView = UIView
+public typealias UIViewControllerOrNSViewController = UIViewController
+public typealias UIHostingControllerOrNSHostingController = UIHostingController
+public typealias UIViewRepresentableOrNSViewRepresentable = UIViewRepresentable
+public typealias UILayoutPriorityOrNSLayoutConstraintPriority = UILayoutPriority
```

Comment:
suggested naming to remove how verbose this is
```suggestion
public typealias PlatformView = UIView
public typealias PlatformViewController = UIViewController
public typealias PlatformHostingController = UIHostingController
public typealias PlatformViewRepresentable = UIViewRepresentable
public typealias PlatformLayoutPriority = UILayoutPriority
```

Reply:
Sure, works for me

Reply:
I think @brynbodayle's suggestion `ViewType`, `ViewControllerType` is nice

Reply:
oh yeah that's better